src/components/HURUmap/Panel/useStyles.js

```diff
@@ -0,0 +1,71 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(
+  ({ typography, palette, transitions, widths, zIndex }) => ({
+    root: {
+      display: "flex",
+      height: "100%",
+      scrollBehavior: "smooth",
+    },
+    drawerOpen: {
+      minWidth: typography.pxToRem(1049),
+      maxWidth: "max-content",
```

Comment:
this is so that the side can stretch to accommodate the screen size (if fixed everything gets pulled to the side), the main content area remains the same regardless (`800px`) @kilemensi